playbook.yml

```diff
@@ -11,161 +11,223 @@
+      force: yes
+      state: link
+
+  - name: SSM Group
+    group:
+      name: SSM
+      system: yes
+
+  - name: Create wherescape user
+    user:
+      name: wherescape
+      group: SSM
+
+  - name: Yum Install Dependencies
+    yum: name={{ item }}
+    with_items:
+      - epel-release
+      - python-pip
+      - gcc
+      - pcre-devel
+      - openssl-devel
+      - zlib-devel
+      - wget
+      - git
+      - cloud-utils
+
+  - name: Pip Install Dependencies
+    pip: 'name={{ item }}'
+    with_items:
+      - virtualenv
+      - AWSCLI
+      - ftputil
+
+  - name: Install Python 2.7
+    shell: '{{ item }}'
+    args:
+      chdir: /usr/src
+    with_items:
+      - wget https://www.python.org/ftp/python/2.7.10/Python-2.7.10.tgz
+      - tar xzf Python-2.7.10.tgz
+
+  - name: Configure Python 2.7
+    shell: ./configure
+    args:
+      chdir: /usr/src/Python-2.7.10
+
+  - name: GET config files from DQ-config-bucket
+    shell: '{{ item }}'
+    with_items:
+      - aws configure set aws_access_key_id '{{user_access_key}}'
+      - aws configure set aws_secret_access_key '{{secret_access_key}}'
+      - aws s3 cp s3://dq-config-bucket/dq-data-ingest-linux-server/ /tmp/install/ --recursive
+
+  - name: Install Postgresql 9.6
+    shell: '{{ item }}'
+    with_items:
+      - rpm -ivh https://yum.postgresql.org/9.6/redhat/rhel-7.3-x86_64/pgdg-centos96-9.6-3.noarch.rpm
+      - yum update -y
+      - yum install -y postgresql96
+
+  - name: ClamAV Install Packages
+    shell: '{{ item }}'
+    with_items:
+      - yum -y install clamav-server clamav-data clamav-update clamav-filesystem clamav clamav-scanner-systemd clamav-devel clamav-lib clamav-server-systemd
+
+  - name: ClamAV SELinux configuration
+    shell: '{{ item }}'
+    with_items:
+      - setenforce 0
+      - setsebool -P antivirus_can_scan_system 1
+      - setsebool -P clamd_use_jit 0
+
+  - name: ClamAV configuration
+    shell: '{{ item }}'
+    with_items:
+      - yes | cp /tmp/install/scan.conf /etc/clamd.d/scan.conf
+      - cp /etc/clamd.d/scan.conf /etc/clamd.d/scan.conf.backup
+      - cp /etc/clamd.d/scan.conf /etc/clamd.conf
+
+#      - sed -i -e "s/^Example/#Example/" /etc/clamd.d/scan.conf
+#      - cp /usr/share/clamav/template/existing clamd.conf /etc/clamd.d/clamd.conf
+#      - chmod -u wherescape 755 /var/lib/clamav/
+#      - name: Add Wherescape to virusgroup
+#      - usermod -g virusgroup wherescape
+#      - name: Add Wherescape to /var/run/
+
+  - name: Freshclam configuration
+    shell: '{{ item }}'
+    with_items:
+      - cp /etc/freshclam.conf /etc/freshclam.conf.bakup
+      - sed -i -e "s/^Example/#Example/" /etc/freshclam.conf
+      - cp /tmp/install/freshclam.service /usr/lib/systemd/system/freshclam.service
+      - freshclam
+
+  - name: Cron Freshclam
+    cron:
+      user: wherescape
+      minute: "0"
+      hour: "01,13"
+      day: "*"
+      job: "/usr/bin/freshclam --quiet"
+
+  - name: Start, Enable Freshclam
```

Comment:
use the service module http://docs.ansible.com/ansible/latest/service_module.html

Reply:
updated